src/test/scala/io/iohk/ethereum/transactions/PendingTransactionsManagerSpec.scala

```diff
@@ -115,14 +115,17 @@ class PendingTransactionsManagerSpec extends FlatSpec with Matchers with ScalaFu
     pendingTransactionsManager ! AddOrOverrideTransaction(firstTx)
     peerManager.expectMsg(PeerManagerActor.GetPeers)
     peerManager.reply(Peers(Map(peer1 -> Handshaked)))
+    Thread.sleep(Timeouts.normalTimeout.toMillis)
```

Comment:
I think yes, I was debugging this test for few hours, after changes in EvmConfig.scala and in package.scala this test was failing